src/RobotHub.cpp

```diff
@@ -7,6 +7,13 @@
 
 namespace rtt::robothub {
 
+constexpr int DEFAULT_GRSIM_FEEDBACK_PORT_BLUE_CONTROL = 30011;
+constexpr int DEFAULT_GRSIM_FEEDBACK_PORT_YELLOW_CONTROL = 30012;
+constexpr int DEFAULT_GRSIM_FEEDBACK_PORT_CONFIGURATION = 30013;
+
+constexpr float DEFAULT_CHIPPER_ANGLE = 45.0f;
+constexpr float MAX_DRIBBLER_SPEED = 1021.0f;
+
```

Comment:
^ Are these two lines for GRSIM? If so, please prepend with GRSIM_ or something.

Reply:
These variables describe properties of our physical robots. These are indeed used in commands for simulators. These values were used in previous year, so I do not know the correctness of them.

I changed the names a bit and added comments. I think that should be sufficient for now at least